tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -911,8 +911,9 @@ Here are some examples of custom effects:
 Ghost
 ~~~~~
 
-::
-
+.. tabs::
+ .. code-tab:: gdscript GDScript
+ 
```

Comment:
```suggestion

```

Reply:
I don't get it, you suggested to delete the line between the code-tab and the code itself?